layersvt/api_dump.h

```diff
@@ -441,6 +441,11 @@ class ApiDumpSettings {
             stream() << "[\n";
         }
 
+        if (use_spaces)
```

Comment:
This was intentional because setfill stays active for basically the duration of the stream, and isn't invalidated. This prevents the need from calling setfill every time a fill is required. 

However, it would be prudent to determine if that behavior is 1. guaranteed (I read a bit of cppreference but have forgotten the specifics) and 2. whether the output of api_dump maintains space/tab consistency.